.github/workflows/pr.yml

```diff
@@ -37,7 +37,7 @@ env:
 
 jobs:
   setup:
-    runs-on: ubuntu-20.04
+    runs-on: "ubuntu-latest"
```

Comment:
Could you bump to the latest ubuntu version string? "latest" breaks builds eventually.

Reply:
Updated.